dScripts/ai/AG/AgBusDoor.cpp

```diff
@@ -23,13 +23,13 @@ void AgBusDoor::OnProximityUpdate(Entity* self, Entity* entering, std::string na
 	m_Counter = 0;
 	m_OuterCounter = 0;
 
-	for (const auto& pair : proximityMonitorComponent->GetProximityObjects("busDoor")) {
-		auto* entity = Game::entityManager->GetEntity(pair.first);
+	for (const auto id : proximityMonitorComponent->GetProximityObjects("busDoor")) {
+		const auto entity = Game::entityManager->GetEntity(id);
```

Comment:
we use auto* for deduced pointers

Reply:
the const auto* const shall make its return i suppose